src/common/utils/connections.ts

```diff
@@ -191,7 +191,10 @@ export const fromConnectionArgs = (
     take = maxTake
```

Comment:
`take` shouldn't greater than `before - 1`.

Reply:
This should get handled in this line: https://github.com/thematters/matters-server/pull/3994/files/9c6549e7c18b49552172aa87e063f9ce7da98e01#diff-b3696ff280de4972964b5a9c4a52ebadd505425e2d39dedca2c79d6f0e113193R196

Reply:
https://github.com/thematters/matters-server/pull/3994/files#r1646243885

Reply:
got it, now I understood. addressing this in https://github.com/thematters/matters-server/pull/3994/files#diff-b3696ff280de4972964b5a9c4a52ebadd505425e2d39dedca2c79d6f0e113193R195-R210